publish-release.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+VERSION="v1"
+CLOUD_PROJECT="prshim"
+
+npm install
+gulp build
+
+# Log in to gcloud with service account or manually with one of the following
+# commands:
+
+# gcloud auth activate-service-account --key-file client-secret.json
+# gcloud auth login
+
+# Ensure gcloud is pointing to the correct project
+gcloud config set project $CLOUD_PROJECT
+
+# Commit projects to the repo
+gsutil cp build/* "gs://$CLOUD_PROJECT/$VERSION/"
+gsutil acl set public-read "gs://$CLOUD_PROJECT/$VERSION/*"
```

Comment:
Check the gsutil help data but I think you can use -r at the bucket level and that will recursively set the desired ACL throughout the entire tree.